src/MatrixFields/field_matrix_iterative_solver.jl

```diff
@@ -375,6 +376,7 @@ end
 function StationaryIterativeSolve(;
     P_alg = nothing,
     n_iters = 1,
+    debug = false,
```

Comment:
Can you add a dosctring explaning what `debug` does and when one would use it?

Reply:
Good idea, done!